src/main/java/com/conversions/HexadecimalToDecimal.java

```diff
@@ -0,0 +1,37 @@
+package com.conversions;
+
+
+public class HexadecimalToDecimal {
+
+	/**
+     * This method converts a Hexadecimal number to a decimal  number
+     *
+     * @param hexadecimalStr
+     * @return decimal number
+     */
+	 public String hexToDecimal(String hexaStr) {
+		 String hexaNumbers = "0123456789ABCDEF";
```

Comment:
Your code fails in case of "AB.CD"

Reply:
because from the point you must multiply for 1/16, the potency start to be negative

Reply:
Yes, that's what I did, thank you